changelog/add-6355-changelog

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Improve disputes list page by removing Disputed on date column by default, add Action column with clear call to action button, highlight urgent disputes' due dates, and color code disputes' statuses.
```

Comment:
I've suggested minor changes to this, feel free to take or leave 👍

```suggestion
Improve disputes list page by hiding the "Disputed on" column by default, add an "Action" column with clear call to action button, highlight urgent disputes' due dates, and color code disputes' statuses.
```

Reply:
Take!